### PR TITLE
Add checklist: How to release Epinio

### PR DESCRIPTION
**Pull request by manno**

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# How to Release Epinio
+
+First release epinio/epinio
+* [ ] check CI is green in epinio/epinio
+* [ ] push version tag (vX.X.X)
+* [ ] wait for release, then edit/check release notes on Github
+
+Now relesease both epinio/helm-charts
+* [ ] look for updatecli PR, created by epinio/epinio release
+* [ ] add commit with new `version:` keys
```
Comment (jimmykarily) [inline docs/release.md:10]:
```suggestion
```

https://github.com/epinio/helm-charts/pull/74

File: docs/release.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* [ ] check that `epinioChart:` in chart/epinio-installer/values.yaml is correct
```
Comment (jimmykarily) [inline docs/release.md:11]: same for the `epinio` chart?

File: docs/release.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+* [ ] wait for chart CI to finish
+* [ ] update Git submodule in epinio/epinio to point to the tested helm-charts repo
```
Comment (jimmykarily) [inline docs/release.md:13]: This is not necessary. We can bump the git submodule at any time. The epinio CI is using chartmuseum to build and use any version currently checked.

**Review by jimmykarily — DISMISSED** [review]

File: docs/release.md (hunk 3 above)
Comment (manno) [reply docs/release.md:13]: There was some discussion if this would affect tests in epinio/epinio.

File: docs/release.md (hunk 2 above)
Comment (manno) [reply docs/release.md:11]: The update didn't work the last time

**Comment by manno** [comment]
moving this to the wiki

**Pull request closed without merging**